Action.c

```diff
@@ -491,12 +499,12 @@ static Htop_Reaction actionHelp(State* st) {
    for (int i = 0; helpLeft[i].key;  i++) { mvaddstr(9+i, 0,  helpLeft[i].key); }
    for (int i = 0; helpRight[i].key; i++) { mvaddstr(9+i, 40, helpRight[i].key); }
    attrset(CRT_colors[PROCESS_THREAD]);
-   mvaddstr(16, 32, "threads");
-   mvaddstr(17, 26, "threads");
+   mvaddstr(17, 32, "threads");
+   mvaddstr(18, 26, "threads");
    attrset(CRT_colors[DEFAULT_COLOR]);
 
    attrset(CRT_colors[HELP_BOLD]);
-   mvaddstr(23,0, "Press any key to return.");
+   mvaddstr(24,0, "Press any key to return.");
```

Comment:
More a long-term issue, but could we _please_ automate these lines?

Reply:
Yes, I'll take a look at it.